src/components/settings/CertificateSection.vue

```diff
@@ -1,92 +1,87 @@
-                                    :href="certificateBlobURL"
-                                    download="certificate.pem"
-                                >
-                                    Download
-                                </a>
-                            </div>
-                            <textarea
-                                id="certificate"
-                                v-model="certificate"
-                                class="form-input input-text text-xs w-full"
-                                rows="15"
-                                readonly
-                            />
+            <div v-else class="lg:flex lg:space-x-12 lg:space-y-0 space-y-4 w-full">
+                <div class="w-full">
+                    <div v-if="hasCertificate">
+                        <div class="flex space-x-4">
+                            <label class="input-label-tmp">Security Certificate</label>
+                            <a
+                                id="downloadCertificate"
+                                :href="certificateDownloadURL"
+                                class="text-sm navigation-link-tmp"
+                                download="certificate.pem"
+                            >Download</a>
                         </div>
+                        <textarea id="certificate" v-model="certificate" rows="19" readonly class="w-full input-text-tmp font-mono" />
+                    </div>
+                    <div v-else>
+                        <span class="block text-sm text-gray-500"> There are currently no certificates associated with your account. </span>
+                        <button id="createCertificate" class="mt-4 w-48 btn-tmp" @click="createCertificate()">Create Certificate</button>
                     </div>
-                </div>
-                <div v-else class="w-full flex flex-col">
-                    <label class="text-gray-700 mb-3">You do not have a UC4 certificate, yet.</label>
-                    <button id="createCertificate" class="btn btn-blue-primary w-48" @click="getCertificate()">Create certificate</button>
                 </div>
             </div>
-            <loading-spinner v-else />
         </div>
-    </section>
+    </BaseSection>
 </template>
 
 <script lang="ts">
+    import BaseSection from "@/components/common/section/BaseSection.vue";
     import { onBeforeMount, ref } from "vue";
     import { useStore } from "@/use/store/store";
-    import LoadingSpinner from "@/components/common/loading/Spinner.vue";
 
     export default {
         name: "CertificateSection",
-        components: { LoadingSpinner },
+        components: {
+            BaseSection,
+        },
         setup() {
-            const busy = ref(false);
-            const certificate = ref("");
-            let certificateBlobURL = ref("");
+            const store = useStore();
+            const isLoading = ref(true);
```

Comment:
I personally disagree and don't see a problem; it's a matter of taste. Why should we initialize it with a different value just to set it in the next line? One would also use available constructors in other languages, e.g. `o = new Object("hello");` instead of going `o = new Object(); o.msg = "hello";`